core/src/main/java/hudson/Functions.java

```diff
@@ -2301,4 +2301,14 @@ public static boolean isContextMenuVisible(Action a) {
             return true;
         }
     }
+
+    /**
+     * Get Locale Display Language depending on the Request's Locale
+     * @param request @{@link HttpServletRequest}
+     * @return @{@link String} Locale Display Language
```

Comment:
The CI-build shows the following error for these lines:

```
[2020-05-27T09:23:25.697Z] [ERROR] /home/jenkins/workspace/Core_jenkins_PR-4751/core/src/main/java/hudson/Functions.java:2307: error: no tag name after @
[2020-05-27T09:23:25.697Z] [ERROR]      * @param request @{@link HttpServletRequest}
[2020-05-27T09:23:25.697Z] [ERROR]                       ^
[2020-05-27T09:23:25.697Z] [ERROR] /home/jenkins/workspace/Core_jenkins_PR-4751/core/src/main/java/hudson/Functions.java:2308: error: no tag name after @
[2020-05-27T09:23:25.697Z] [ERROR]      * @return @{@link String} Locale Display Language
```

I think that there a few @ signs should be removed.

```suggestion
     * @param request {@link HttpServletRequest}
     * @return {@link String} Locale Display Language
```